gatsby/util/append-trailing-slash.js

```diff
@@ -0,0 +1,7 @@
+const appendTrailingSlash = (path) => {
```

Comment:
having all those isolated files brings testing back to the table and we can add that either with this refactoring PR or in a later PR. I think testing the gatsby code is currently more important than testing the components on our website.

Reply:
Yes, those isolated files encourage a good architecture and design - which in turn makes testing quite simple. I will add a test for this one.

Reply:
I also added a test for `build-gatsby-cloud-preview-url.js`